apps/dashboard/public/noVNC-1.3.0/app/ui.js

```diff
@@ -955,7 +955,8 @@ const UI = {
             .then(() => navigator.clipboard.readText())
             .then((clipboardText) => {
                 const text = document.getElementById('noVNC_clipboard_text').value;
-                if (clipboardText !== text) {
+                const nlClipboardText = clipboardText.replace(/\r\n/g, "\n");
+                if (clipboardText !== text && nlClipboardText !== text) {
```

Comment:
Thinking about this a little more, can we just cast `text` like so:

```javascript
var text = document.getElementById('noVNC_clipboard_text').value;
text = String(text).replace(/\r\n/g, "\n");
```

That way we don't have a second variable to compare against, and with the casting to string it provides a little safety against null or undefined values.

Reply:
It's not `text` that contains the carriage returns but `clipboardText` coming in from the local clipboard. The following seems to work though for simplifying it:
``` js
.then((clipboardText) => {
    clipboardText = clipboardText.replace(/\r\n/g, "\n");
    const text = document.getElementById('noVNC_clipboard_text').value;
    if (clipboardText !== text) {
```